docs/importing_models.md

```diff
@@ -0,0 +1,18 @@
+# Introduction {#openvino_docs_importing_models_introduction}
```

Comment:
I think that this should be a content of `documentation.md` if that document is really an intro to model importing.

Reply:
documentation.md is just a quick overview of what is included in the Documentation tab, it's not the proper place to provide intro text and graph